src/App.js

```diff
@@ -9,7 +12,7 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <Article {...article} />
```

Comment:
You can use spread operator here, but it's not clear what properties are you passing to this component, let's do it in more clear way, like:
```
  <Article 
  	key={value}
		anotherKey={anotherValue}
		and so on...
  />
```